src/UncontrolledCarousel.js

```diff
@@ -60,11 +61,15 @@ class UncontrolledCarousel extends Component {
     const { activeIndex } = this.state;
 
     const slides = items.map((item) => {
+      const key = item.key || item.src;
+      if (!item.key) {
+        warnOnce('Item in UncontrolledCarousel is missing a key. Please provide a unique key to the item to avoid rendering react children with duplicate keys.');
```

Comment:
I don't think this is really needed as multiple items with the same image in a carousel does not seem typical. Simply providing the ability to specify and override src with key should be good enough.

Reply:
81b6c5c2c9dfbedd0e483983976137ef18241b17